youtube_transcript_api/_errors.py

```diff
@@ -118,3 +134,38 @@ def cause(self):
             requested_language_codes=self._requested_language_codes,
             transcript_data=str(self._transcript_data),
         )
+
+
+def get_playability_error(playability_json):
+    """
+    Using the json extracted from playabilityStatus,
+    returns a custom error based on the value of the "status" key.
+    
+    Anything that is not {"status": "OK"} is likely an error.
+    """
+    reason = playability_json.get("status")
+    if reason == 'LOGIN_REQUIRED':
+        # error for age related playability
```

Comment:
`LOGIN_REQUIRED` also happens when you need to "Sign in to confirm you’re not a bot". This simpler pull request addresses this: https://github.com/jdepoix/youtube-transcript-api/pull/337